engine/access/rpc/connection/connection_test.go

```diff
@@ -50,6 +52,12 @@ func TestProxyAccessAPI(t *testing.T) {
 		unittest.Logger(),
 		connectionFactory.AccessMetrics,
 		0,
+		CircuitBreakerConfig{
+			Enabled:        false,
+			MaxFailures:    0,
+			MaxRequests:    0,
+			RestoreTimeout: 0,
+		},
```

Comment:
nit, add a method that returns this default disabled config and call that instead of repeating this struct.

Alternatively, since 0 and false are default initial values, so you could just use
```golang
CircuitBreakerConfig{}
```